conda/rapids-tpcx-bb.yml

```diff
@@ -8,20 +9,22 @@ dependencies:
   - python=3.7
   - cudatoolkit=10.2
   - cudf
+  - rmm
   - dask-cuda
   - dask-cudf
   - cuml
-  - scipy
   - ucx-py
   - ucx-proc=*=gpu
+  - blazingsql
+  - scipy
   - scikit-learn
   - cupy=7
   - spacy
   - oauth2client
   - asyncssh
   - psutil
-  - jupyterlab
   - ipykernel
+  - jupyterlab
```

Comment:
Do we need this?

Reply:
We don't strictly need it, but it's convenient and we aren't too concerned about dependency bloat at this point. Can be removed if desirable

Reply:
Keeping for now, but can always remove.